Dockerfile

```diff
@@ -6,5 +6,6 @@ RUN hack/build-go.sh
 FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
 COPY --from=builder /go/src/github.com/openshift/cluster-version-operator/_output/linux/amd64/cluster-version-operator /usr/bin/
 COPY install /manifests
+COPY vendor/github.com/openshift/api/config/v1/*_cluster-version-operator_*.crd.yaml /manifests/
```

Comment:
I would rather like this to be explicit, rather than regex.

Reply:
fair enough. changing.

Reply:
done